contracts/src/v0.8/automation/dev/interfaces/v2_3/IAutomationRegistryMaster2_3.sol

```diff
@@ -73,10 +76,10 @@ interface IAutomationRegistryMaster2_3 {
     bytes offchainConfig
   );
   event DedupKeyAdded(bytes32 indexed dedupKey);
+  event FeesWithdrawn(address indexed recipient, address indexed assetAddress, uint256 amount);
   event FundsAdded(uint256 indexed id, address indexed from, uint96 amount);
   event FundsWithdrawn(uint256 indexed id, uint256 amount, address to);
   event InsufficientFundsUpkeepReport(uint256 indexed id, bytes trigger);
-  event OwnerFundsWithdrawn(uint96 amount);
```

Comment:
can we keep this event? if not, it won't be compatible with existing atlas.

@cmalec what do you think?